OptionSelector/CQCategory.h

```diff
@@ -0,0 +1,18 @@
+//
+//  CQCategory.h
+//  OptionSelector
+//
+//  Created by Henna on 8/10/15.
+//  Copyright (c) 2015 Mike Kavouras. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@interface CQCategory : NSObject
+
+    @property (nonatomic) NSString * name;
```

Comment:
Style: These shouldn't be indented (they should line up with the `@` in the `@interface` directive.